metadata/movies/movies.yml

```diff
@@ -308,6 +314,9 @@ metadata:
   36648:
     title: "Blade Trinity"
     url_poster: https://github.com/jade-team/plex-meta-manager-config/raw/main/metadata/movies/assets/36648-blade-trinity.jpg
+  38700:
+    title: "Bad Boys for life"
```

Comment:
¿`Dos policías rebeldes 3 (Bad Boys for Life)`?

Reply:
Done